ports/x264/portfile.cmake

```
@@ -1,6 +1,6 @@
 set(X264_VERSION 157)
 
-vcpkg_fail_port_install(ON_TARGET "Linux" "OSX")
+#vcpkg_fail_port_install(ON_TARGET "Linux" "OSX") 
```

```suggestion
vcpkg_fail_port_install(ON_TARGET "OSX") 
```

Based on the baseline, I think x264 still does not support osx.